prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudication-container/adjudication-container.component.ts

```diff
@@ -81,6 +83,13 @@ export class AdjudicationContainerComponent implements OnInit {
   public onNotify(enrolleeId: number) {
     this.adjudicationResource.getEnrolleeById(enrolleeId)
       .pipe(
+        exhaustMap((enrollee: HttpEnrollee) => {
+          if (enrollee.contactEmail) {
+            return of(enrollee);
+          }
+          this.toastService.openErrorToast('Enrollee does not contain a Contact Email');
```

Comment:
Should we maybe say 'Enrollee does not have a Contact Email' instead of 'contain' since this is displayed to the user?

Reply:
Also, don't think "Contact Email" needs to be capitalized